### ISSUE TITLE
task completed/failed should have diff't noises

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#162347]
1. play the task completion and failure noises via `Help -> List Audio Cues`
2. turn on `editor.accessibilitySupport`
3. run a task which succeeds (0 exit code) and hear the completion noise
4. run a task which fails (non-zero exit code) and hear the failure noise
